Controls/Chart/Chart.UWP/Engine/Axes/AxisModel.cs

```diff
@@ -102,6 +103,21 @@ public IContentFormatter ContentFormatter
         }
 
         /// <summary>
+        /// Gets or sets the <see cref="ILabelCreator"/> instance used to decide whether Label should be created.
+        /// </summary>
+        public ILabelCreator LabelCreator
```

Comment:
I tried to put it into the ChartSeriesLabelDefinition class. Event though the class is public  it is initialized and used only internally and couldn't think of any other easy way how to users could implement their logic about which label should be created. 

I believe the approach of introducing such interface where they can implement their logic is much more straightforward for the users.

About the name as SpriteVisual instances are created I named it factory. Any suggestions about another name is appreciated. 😄